src/app/pages/press/mobile-selector/mobile-selector.js

```diff
@@ -1,45 +1,55 @@
-import $ from '~/helpers/$';
-import {Controller} from 'superb.js';
-import {on} from '~/helpers/controller/decorators';
-import {description as template} from './mobile-selector.html';
-import css from './mobile-selector.css';
+import React, {useState} from 'react';
+import WrappedJsx from '~/controllers/jsx-wrapper';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import './mobile-selector.css';
 
-export default class MobileSelector extends Controller {
+function MenuItem({selectedValue, value, onChange}) {
+    const selectedClass = value === selectedValue ? 'selected' : null;
 
-    init(getProps, onChange) {
-        this.template = template;
-        this.getProps = getProps;
-        this.onChange = onChange;
-        this.view = {
-            classes: ['mobile-selector']
-        };
-        this.css = css;
-        this.model = () => this.getModel();
-        this.showingMenu = false;
+    function onClick() {
+        onChange(value);
     }
 
-    getModel() {
-        this.props = this.getProps();
+    return (
+        <div role="menuitem" className={selectedClass} onClick={onClick}>{value}</div>
+    );
+}
 
-        return {
-            selectedValue: this.props.selectedValue,
-            values: this.props.values,
-            showingMenu: this.showingMenu,
-            selectedClass: (v) => v === this.props.selectedValue ? 'selected' : null
-        };
-    }
+function Menu({selectedValue, values, onChange}) {
+    return (
+        <div className="fixed-overlay">
+            <div role="menu">
+                {
+                    values.map((value) =>
+                        <MenuItem selectedValue={selectedValue} value={value} onChange={onChange}/>
```

Comment:
map needs `key` 😁 I also sometimes will use this type of prop if the variable names match properties:
` <MenuItem {...{ selectedValue, onChange, value }} />`